libbeat/docs/outputconfig.asciidoc

```diff
@@ -608,7 +608,7 @@ The password for connecting to Kafka.
 ===== `topic`
 
 The Kafka topic used for produced events. The setting can be a format string
-using any event field. To set the topic from document type use `%{[type]}`.
+using any event field. You can create a custom field in the prospector with `fields_under_root: true` and reference back to it here.
```

Comment:
I think the custom field doesn't need to be under root, right? It can be referenced via the dotted notation. Just wondering if you've found a limitation regarding that.

Reply:
I'd expect that the dotted notation works fine. It's just that we didn't test that.